lib/ruby-lint/analysis/loop_keywords.rb

```diff
@@ -57,7 +58,7 @@ class LoopKeywords < Base
       # @param [RubyLint::AST::Node] node
       #
       def verify_keyword(keyword, node)
-        if current_scope.type != :block and !@allow_keyword
+        if current_scope.type != :block && [0, nil].include?(@allow_keyword)
```

Comment:
I have fixed that, renaming the attribute to loop_nesting, in https://github.com/mvidner/ruby-lint/commit/d38dc7d2f1d62085c02a592ae9a5a79a2102ef52 (branch https://github.com/mvidner/ruby-lint/tree/fix_nested_loops).